content/en/docs/reference/feature-gates.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Feature Gates
-content_template: templates/concept
 ---
 
-{{% capture overview %}}
+{% capture overview %}
 This page contains an overview of the various feature gates an administrator
 can specify on different Kubernetes components.
-{{% /capture %}}
+{% endcapture %}
```

Comment:
/assign @chenopis 

I'm not sure the right way to do this in Hugo yet.

Reply:
Verified that the way in red in line 9 is the new right way, so please revert these back to the Hugo way.